packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -152,7 +154,8 @@ async function redeemSharingLink(
 
 async function fetchLatestSnapshotCore(
     odspResolvedUrl: IOdspResolvedUrl,
-    storageTokenFetcher: (options: TokenFetchOptions, name: string) => Promise<string | null>,
+    storageTokenFetcher: (options: TokenFetchOptions, name: string, alwaysRecordTokenFetchTelemetry?: boolean) =>
```

Comment:
It seems like there's a bunch of these
```
 storageTokenFetcher: (options: TokenFetchOptions, name: string, alwaysRecordTokenFetchTelemetry?: boolean) =>
        Promise<string | null>
```
lines.

Would it make sense to create a type and use it throughout?

```
type StorageTokenFetcherCallback = (options: TokenFetchOptions, name: string, alwaysRecordTokenFetchTelemetry?: boolean) =>
    Promise<string | null>;

<...>

storageTokenFetcher: StorageTokenFetcherCallback